lms/static/scripts/frontend_apps/components/LMSFilePicker.js

```diff
@@ -25,13 +27,14 @@ import FileList from './FileList';
  *
  * @param {NoFilesMessageProps} props
  */
-function NoFilesMessage({ href }) {
+function NoFilesMessage({ href, inSubfolder }) {
+  const documentContext = inSubfolder ? 'folder' : 'course';
   return (
     <div className="FileList__no-files-message">
       <p>
-        There are no PDFs in this course.{' '}
+        There are no PDFs in this {documentContext}.{' '}
```

Comment:
When we come to add support for translations we'll have to make the entire sentence conditional on the context, not just the word, because in non-English languages the choice of noun can alter other words in the sentence (eg. due to word gender).